packages/components/src/date-time/index.js

```diff
@@ -25,10 +26,10 @@ function DateTimePicker(
 		currentDate,
 		is12Hour,
 		isInvalidDate,
-		onMonthPreviewed,
 		onChange,
 		events,
 		keepOpen,
+		onMonthChange = noop,
```

Comment:
Why did you change the prop here? onMonthChange suggests that the actual value of the month changed, is this the case?

Reply:
yes, it partially is. The thing is react-dates component doesn't have a callback to detect when the month changes. This custom callback runs when the month changes by clicking on the next/prev button. It isn't ideal but works for this purpose.

Reply:
but why not just use the previous onMonthPreviewed?

Reply:
> but why not just use the previous onMonthPreviewed? 

I think `onMonthChange` name fits better according to what the callback does. I can revert it, though 👍 

> AFAIK clicking next/prev doesn't actually change the value

If with `value you mean to the `currentDate` time, that's correct. For this reason, we store the `currentMonth` in a different state.

Reply:
>I think onMonthChange name fits better according to what the callback does. I can revert it, though 👍

This is an unnecessary breaking change for me. and for me "change" means the actual value changed and not just what's currently being shown so I do prefer the previous name.

Reply:
marking as solved because https://github.com/WordPress/gutenberg/pull/29716/commits/d2d3758e9d61e2a1705979a37e26f7bfbeae9339 and https://github.com/WordPress/gutenberg/pull/29716/commits/621f3e42045afd87bab7362410ad975ac2ecdd22